app/providers/list/page.tsx

```
@@ -0,0 +1,127 @@
+    const res = await fetch(
+      `${baseUrl}/providers?page=${page}&pageSize=${pageSize}`,
+      { next: { revalidate: 60 } }
+    );
+    
+    if (!res.ok) {
+      throw new Error('Failed to fetch providers list');
+    }
+    
+    const data = await res.json();
+    // Map snake_case API fields to camelCase for the client component
+    const processedProviders = data.providers.map((provider: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)